src/components/AddItemForm.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { withFirestore } from 'react-firestore';
+import '../CSS/AddItemForm.css';
+class AddItemForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      next_purchase: null,
+      last_purchase: null,
+      user_token: 'shpongle',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSchedule = this.handleSchedule.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleSchedule(event) {
+    this.setState({ next_purchase: event.target.value });
+  }
+
+  handleSubmit(event) {
+    this.props.firestore
+      .collection('itemform')
+      .add(this.state)
+      .then(function(docRef) {
+        console.log('Document written with ID: ', docRef.id);
+      })
+      .catch(function(error) {
+        console.error('Error adding document:', error);
+      });
+    event.preventDefault();
+    //To reset the input field after the user hits submit
+    this.setState({
+      name: '',
+      ext_purchase: null,
+      last_purchase: null,
+      user_token: 'shpongle',
+    });
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label class="name">
+          Name:
+          <input
+            type="text"
+            value={this.state.name}
+            onChange={this.handleChange}
+          />
+        </label>
+        <br />
```

Comment:
❤️ this semantic and accessible form 😍

Alternatively, you could remove the `<br />` tags and use css:

```css
#addItemForm label {
  display: block;
}
```